yarn.lock

```diff
@@ -125,7 +125,7 @@
     supports-color "^5.5.0"
     tslib "^1.9.3"
 
-"@heroku-cli/command@8.2.1", "@heroku-cli/command@^8.1.17", "@heroku-cli/command@^8.2.0", "@heroku-cli/command@^8.2.1":
+"@heroku-cli/command@8.2.1":
   version "8.2.1"
```

Comment:
seems that 8.2.1 is still being pulled in

Reply:
@jdxcode strange